src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ CHANGELOG
  * deprecated using the `Bic`, `Country`, `Currency`, `Language` and `Locale` constraints without `symfony/intl`
  * deprecated using the `Email` constraint without `egulias/email-validator`
  * deprecated using the `Expression` constraint without `symfony/expression-language`
+ * validating a BIC based on a given IBAN
```

Comment:
for discovery, id say:

- added options `iban` and `propertyPath` to `Bic` constraint